lectures/scripts/clock_deblur.py

```diff
@@ -2,9 +2,9 @@
 import matplotlib.pyplot as plt
 
 from skimage import io
-from skimage.viewer import ImageViewer
-from skimage.viewer.widgets import Slider
-from skimage.viewer.plugins.base import Plugin
+#from skimage.viewer import ImageViewer
+#from skimage.viewer.widgets import Slider
+#from skimage.viewer.plugins.base import Plugin
```

Comment:
```suggestion
```

Reply:
I hesitated to remove being afraid that the logic would be difficult to understand afterwards. No strong feelings through.